packages/strapi-plugin-users-permissions/controllers/User.js

```diff
@@ -52,6 +52,18 @@ module.exports = {
     ctx.send(data);
   },
 
+  /**
+   * Retrieve user count.
+   * @return {Number}
+   */
+
+  async count(ctx) {
+    if (ctx.query._q) {
+      return await strapi.plugins['users-permissions'].services.user.countSearch(ctx.query);
```

Comment:
You didn't implement this method in the service :)

Reply:
I'm sorry, totally forgot to add it!